backend/controller/auth.controller.js

```diff
@@ -300,7 +299,8 @@ const signOut = asyncHandler(async (req, res, next) => {
     httpOnly: true,
     maxAge: new Date().now, //  current date
     path: "/",
-    sameSite: "Lax",
+    sameSite: process.env.NODE_ENV === "production" ? "None" : "Lax",
+    secure: process.env.NODE_ENV === "production" ? true : false,
```

Comment:
Have you tested this locally? Not in postman but in browser as well?

Reply:
tested